### ISSUE TITLE
bin/sqelf: No such file or directory

### ISSUE THREAD

**Issue by adamshergold** [datalust/seq-input-gelf#120]
Hi,

The docker compose example isn't working for me (latest docker desktop, MacOS M1) ... the gelf container won't successfully start:

```bash
seq-seq-input-gelf-1  | /run.sh: line 10: bin/sqelf: No such file or directory
seq-seq-input-gelf-1  | /run.sh: line 10: bin/seqcli/seqcli: No such file or directory
```

Compose file is

```yml
version: '3'
services:
  seq-input-gelf:
    image: datalust/seq-input-gelf:latest
    depends_on:
      - seq
    ports:
      - "12201:12201/udp"
    environment:
      SEQ_ADDRESS: "http://seq:5341"
    restart: unless-stopped
  seq:
    image: datalust/seq:latest
    ports:
      - "5341:80"
    environment:
      ACCEPT_EULA: "Y"
    restart: unless-stopped
    volumes:
      - ./seq-data:/data
```

Full output

```bash
(base) adam@adams-mac-mini seq % docker compose up                      
[+] Running 3/2
 ⠿ Network seq_default             Created                                                                                                      0.0s
 ⠿ Container seq-seq-1             Created                                                                                                      0.0s
 ⠿ Container seq-seq-input-gelf-1  Created                                                                                                      0.0s
Attaching to seq-seq-1, seq-seq-input-gelf-1
seq-seq-input-gelf-1  | /run.sh: line 10: bin/sqelf: No such file or directory
seq-seq-input-gelf-1  | /run.sh: line 10: bin/seqcli/seqcli: No such file or directory
seq-seq-input-gelf-1 exited with code 127
seq-seq-input-gelf-1  | /run.sh: line 10: bin/sqelf: No such file or directory
seq-seq-input-gelf-1  | /run.sh: line 10: bin/seqcli/seqcli: No such file or directory
seq-seq-input-gelf-1 exited with code 127
seq-seq-1             | ────────────────────────────────────────
seq-seq-1             |   Seq ♦ Machine data, for humans.
seq-seq-1             | ─────────── © 2023 Datalust Pty Ltd ────
seq-seq-1             | 
seq-seq-1             | Running as server; press Ctrl+C to exit.
seq-seq-1             | 
seq-seq-input-gelf-1  | /run.sh: line 10: bin/seqcli/seqcli: No such file or directory
seq-seq-input-gelf-1  | /run.sh: line 10: bin/sqelf: No such file or directory
seq-seq-input-gelf-1 exited with code 127
seq-seq-1             | [12:30:07 INF] Seq "2023.1.8876" running on OS "Linux 5.15.49-linuxkit #1 SMP PREEMPT Tue Sep 13 07:51:32 UTC 2022"
seq-seq-1             | [12:30:07 INF] Seq detected 4124.512256 MB of RAM
seq-seq-1             | [12:30:07 INF] Opening event store at "/data/Stream/stream.flare"
seq-seq-1             | [12:30:07 INF] Ingestion enabled
seq-seq-1             | [12:30:07 INF] Opening metastore "/data/Documents/metastore.flare"
seq-seq-1             | [12:30:07 INF] Storage subsystem available
seq-seq-1             | [12:30:07 INF] Seq listening on ["http://localhost/", "https://localhost/", "http://localhost:5341/", "https://localhost:45341/"]
seq-seq-input-gelf-1  | /run.sh: line 10: bin/sqelf: No such file or directory
seq-seq-input-gelf-1  | /run.sh: line 10: bin/seqcli/seqcli: No such file or directory
seq-seq-input-gelf-1 exited with code 127
seq-seq-input-gelf-1  | /run.sh: line 10: bin/sqelf: No such file or directory
seq-seq-input-gelf-1  | /run.sh: line 10: bin/seqcli/seqcli: No such file or directory
seq-seq-input-gelf-1 exited with code 127
^CGracefully stopping... (press Ctrl+C again to force)
Aborting on container exit...
[+] Running 2/2
 ⠿ Container seq-seq-input-gelf-1  Stopped                                                                                                      0.0s
 ⠿ Container seq-seq-1             Stopped                                                                                                      2.3s
canceled
```

Can't see what I'm doing wrong here ... any ideas?

Thanks!

**Comment by nblumhardt** [comment]
Thanks for the report - it looks as though there's still some build work required here, we'll investigate 👍

**Comment by KodrAus** [comment]
Thanks for your patience on this one @adamshergold :bow: This should be fixed as of `datalust/seq-input-gelf:3.0.443`.

**Comment by adamshergold** [comment]
> Thanks for your patience on this one @adamshergold 🙇 This should be fixed as of `datalust/seq-input-gelf:3.0.443`.

Yes, all good now - many thanks!
